### PR TITLE
Add default image for DrinkCase

### PR DESCRIPTION
**Pull request by JasonVinomofo**

### CODE REVIEW

File: app/uploaders/image_uploader.rb [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 
   Attacher.validate do
     validate_max_size 1 * 1024 * 1024,
                       message: 'Image is too large (max is 1 MB)'
     validate_mime_type_inclusion %w(image/jpeg image/jpg image/png image/gif),
                                  message: 'must be JPEG, PNG or GIF'
   end
+
+  Attacher.default_url do
+    'https://mofobottleo-staging.s3-ap-southeast-2.amazonaws.com/store/8a1177368b6bf6da17ea4f026a5e1f6d.png'
```
Comment (daphsta) [inline app/uploaders/image_uploader.rb]: this could be stored as an environment variable

Comment (JasonVinomofo) [inline app/uploaders/image_uploader.rb]: Good point.  Makes sense if I'm going to abstract this app into a template or something.  At the moment, I'm just planning on using in Vinomofo though, and managing more environment variables seems like more work.  Would you say it's better to have an environment variable?  If so, I'll put it in. 👍

Comment (michaeljd) [inline app/uploaders/image_uploader.rb]: You should put `https://mofobottleo-staging.s3-ap-southeast-2.amazonaws.com/store/` in an environment variable, and rename the image to `default_image.png` or something else that describes what the image is.

Comment (michaeljd) [inline app/uploaders/image_uploader.rb]: In this particular case though I think a relative URL would be more relevant?

File: app/views/drink_cases/index.html.erb [hunk 2]
```diff
@@ -18,9 +18,7 @@
   <tr>
     <td class='drink_case-name'>
       <%= link_to drink_case_path(drink_case), id: "show_drink_case_#{ drink_case.id }" do %>
-        <% if drink_case.image.present? %>
-          <%= image_tag drink_case.image_url(public: true), class: 'drink_case-image' %>
-        <% end %>
+        <%= image_tag drink_case.image_url(public: true), class: 'drink_case-image' %>
```
Comment (michaeljd) [inline app/views/drink_cases/index.html.erb]: What does `public: true` do?

Comment (JasonVinomofo) [inline app/views/drink_cases/index.html.erb]: `public: true` removes authentication query params for S3 in the url for the image.

**Comment by JasonVinomofo** [comment]
Yay! thanks

**Pull request merged**